mino/router/tree/mod.go

```diff
@@ -32,8 +31,17 @@ type Router struct {
 	hsFac     router.HandshakeFactory
 }
 
+// RouterOption is the signature of the option constructor
+type RouterOption func(*Router)
+
+func WithHeight(maxHeight int) RouterOption {
```

Comment:
@nkcr Is it possible to add a variable to an option function ?

Reply:
yes, that's the point :)

Reply:
Now that's magic to me!

Reply:
ditto.